src/annotator/sidebar.js

```diff
@@ -140,7 +142,9 @@ export default class Sidebar {
     const toolbarContainer = document.createElement('div');
     this.toolbar = new ToolbarController(toolbarContainer, {
       createAnnotation: () =>
-        this._guestRPC.call('createAnnotationIn', this._guestWithSelection),
+        this._guestRPC.forEach(rpc =>
+          rpc.call('createAnnotationIn', this._guestWithSelection)
+        ),
```

Comment:
A follow-up step here will be to call `createAnnotationIn` only on the specific guest with the selection.